config/prometheus/monitor.yaml

```diff
@@ -0,0 +1,16 @@
+
+# Prometheus Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Does this mean we can remove our explicit creation of the servicemonitor?

Reply:
No this generates a ServiceMonitor to scrape metrics from the cluster-logging-operator container. This is not a replacement for ServiceMonitors CLO reconciles for fluentd for example. Latter need to stay.